compiler/src/iree/compiler/Dialect/Flow/Transforms/ConvertConv2DToImg2Col.cpp

```diff
@@ -169,38 +171,30 @@ class Conv2DImg2ColMatmulConversion
 // input channles so each convolution can be a matrix-vector product and
 // by transposing both input filter so channles are outer most the computation
 // is a batched matrix-vector product.
-class DepthwiseConv2DNHWCHWCImg2ColMatmulConversion
+class ConvertDepthwiseConv2DNhwcHwc final
     : public OpRewritePattern<linalg::DepthwiseConv2DNhwcHwcOp> {
  public:
   using OpRewritePattern<linalg::DepthwiseConv2DNhwcHwcOp>::OpRewritePattern;
 
   LogicalResult matchAndRewrite(linalg::DepthwiseConv2DNhwcHwcOp convOp,
                                 PatternRewriter &rewriter) const override {
-    RankedTensorType inputTensorType =
-        convOp.getInputOperand(0)->get().getType().dyn_cast<RankedTensorType>();
-    RankedTensorType filterTensorType =
-        convOp.getInputOperand(1)->get().getType().dyn_cast<RankedTensorType>();
-    RankedTensorType outputTensorType = convOp.getOutputOperand(0)
-                                            ->get()
-                                            .getType()
-                                            .dyn_cast<RankedTensorType>();
-
-    if (!filterTensorType || !inputTensorType) return failure();
-    if (!filterTensorType.hasStaticShape() || !inputTensorType.hasStaticShape())
+    auto inputType = convOp.getInputs()[0].getType().cast<RankedTensorType>();
+    auto filterType = convOp.getInputs()[1].getType().cast<RankedTensorType>();
+    auto outputType = convOp.getOutputs()[0].getType().cast<RankedTensorType>();
+
+    if (!filterType.hasStaticShape() || !inputType.hasStaticShape())
       return failure();
 
-    // TODO(ataei) : Support padding & dilation.
+    // TODO: Support padding & dilation.
```

Comment:
style nit: add braces on simple single-statement bodies of if condition or make it a single line.

also, let's remove "padding" from TODO